src/conversion_helper.js

```diff
@@ -23,11 +25,12 @@ function distanceMap(m) {
 }
 
 function weightMap(g) {
-  const kg = g/1000;
-  if (g < 1000) {
+  inRangeVal = Math.max(g, inRangeVal);
+  const kg = g/1000, inRangeValKg = inRangeVal/1000;
```

Comment:
Can these constant assignments be broken into two separate lines? I know we don't have a style guideline yet, but this king of triggers me, sorry XD